config/jobs/kubernetes/sig-cloud-provider/azure/release-1.26.yaml

```diff
@@ -8,7 +8,7 @@ presubmits:
     run_if_changed: 'azure.*\.go'
     path_alias: k8s.io/kubernetes
     branches:
-      - release-1.22
+      - release-1.26
```

Comment:
was this auto-generated? Can you please update the README instructions to include 1.26 instead of 1.22 in the command to generate these?

Reply:
+1

Reply:
No, I changed this myself when removing the 1.22 dashboard. Are we supposed to autogenerate this file?